howso/get_cases.amlg

```diff
@@ -88,39 +88,37 @@
 	;returns assoc with features and cases - a list of lists of all feature values. Retrieves all feature values for cases for
 	;all (unordered) sessions in the order they were trained within each session. If a session is specified, only that session's
 	;cases wil be output.
-	;
-	;parameters:
-	; features: list of features to retrieve.
-	; indicate_imputed: if set to 1 will return the audit data of which features for each row were imputed (auto-filled)
-	; session: optional session from which to get cases (even if by condition).
-	; case_indices: optional, list of pair (list) of session id and index, where index is the original 0-based session_training_index of the
-	;		case as it was trained. If specified, ignores session and condition/precision parameters.
-	; condition: optional a query condition describing the cases to return.
-	; num_cases: optional, limit on the number of cases to retrieve; If set to zero there will be no limit.
-	;		If null, will be set to k if precision is "similar" or no limit if precision is "exact". default is null
-	; precision: optional string. default is 'exact', used only with 'condition' parameter, will find exact matches if 'exact' and similar cases if 'similar'.
-	; output_ids: optional, flag. When true only returns an assoc of caseds and no features
-	; skip_decoding: set flag to skip decoding feature values into their nominal values for output.
 	#get_cases
 	(declare
 		(assoc
 			;{type "list" values "string"}
+			;list of features to retrieve.
 			features (list)
 			;{type "number"}
+			;set flag to skip decoding feature values into their nominal values for output.
 			skip_decoding 0
 			;{type "string"}
+			;optional session from which to get cases (even if by condition).
 			session (null)
 			;{ref "CaseIndices"}
+			;optional, list of pair (list) of session id and index, where index is the original 0-based session_training_index of the
+			;	case as it was trained. If specified, ignores session and condition/precision parameters.
 			case_indices (null)
 			;{ref "Condition"}
+			;optional a query condition describing the cases to return.
 			condition (null)
 			;{type "number" min 0}
+			;optional, limit on the number of cases to retrieve; If set to zero there will be no limit.
+			;	If null, will be set to k if precision is "similar" or no limit if precision is "exact". default is null
 			num_cases (null)
 			;{ref "Precision"}
+			;optional string. default is 'exact', used only with 'condition' parameter, will find exact matches if 'exact' and similar cases if 'similar'.
 			precision "exact"
 			;{type "boolean"}
+			;if set to 1 will return the audit data of which features for each row were imputed (auto-filled)
 			indicate_imputed (false)
 			;{type "boolean"}
+			;optional, flag. When true only returns an assoc of caseds and no features
```

Comment:
I figure this should be "case ids"

Reply:
Addressed in d8a9404f922319cfd8e1c60555d514603ff45ec3